app/models/volunteer_op.rb

```diff
@@ -77,8 +77,13 @@ def self.get_source(volunteer_ops)
     source
   end
 
+  def new_volop?
```

Comment:
might we consider calling this method just `new?` since we're calling the method on a volop model ...?

Reply:
Yes, that is a much better name!